oraclizeAPI_0.4.25.sol

```diff
@@ -1080,7 +1080,9 @@ contract usingOraclize {
         internal
         returns (bytes32 id)
     {
-        uint price = oraclize.getPrice(datasource);
+        uint256 price = oraclize.addressCustomPaymentToken(address(this)) == address(0)
```

Comment:
I'm definitely thinking we should have a separate set of these methods, for the case of contracts wanting to support ETH and token payments, for efficiency's sake and to not burden ETH only users. We are agnostic to which payment option is used.

Reply:
Ibid last comment.